packages/tracing/src/browser/browsertracing.ts

```diff
@@ -190,7 +190,6 @@ export class BrowserTracing implements Integration {
 
     // eslint-disable-next-line @typescript-eslint/unbound-method
     const { beforeNavigate, idleTimeout, maxTransactionDuration } = this.options;
-
```

Comment:
Accident or not? No other changes in this file and was not part of the PR originally.